### ISSUE TITLE
Default location for app-specific user data

### ISSUE THREAD

**Issue by xrotwang** [OneAdder/lingtypology#3]
I've seen you are looking in `.lingtypology_data` for user-supplied Glottolog data. There's a python package to determine locations for such data in a fairly cross-platform way: [appdirs](https://pypi.org/project/appdirs/). So in this case, rather than in `~/.lingtypology_data` one would look in `appdirs.user_data_dir('lingtypology')`.

**Comment by OneAdder** [comment]
The problem is that I make users download Glottolog into this directory. It'd better be not too far away

**Comment by xrotwang** [comment]
Yes, it really boils down to whether your users know these default locations for their system (many don't). But on the other hand, being able to point people to the `appdirs` docs is somewhat simpler than writing your own documentation about `.lingtypology_data`.

Overall, I'd say this just strenghtens my point that the most flexible solution would be one, where the user can put the glottolog repos wherever they want - but the API of the package provides a hook to pass this location.

**Comment by OneAdder** [comment]
In this case, I'll have to create a config file that points to the location. And it leads to the start point: where to store the config.  
Or the users will have to manually type the location of the Glottolog data files each time they want to import the package. I wouldn't like to do so (if I were a user).

**Comment by xrotwang** [comment]
But your package is supposed to be used programmatically, right? It only provides an API, so it will only be used from other people's code, which means they will likely have to type the path only once and then re-use, copy-and-paste, etc.  I've been down the path of second-guessing best / most-intuitive locations for config and data and come full-circle to "explicit is better than implicit". If people find typing tedious, they have all sorts of local solutions available, like creating shell aliases, etc.

**Comment by OneAdder** [comment]
Well, I'll think about. Thank you.
